LEGO1/mxdirectdraw.cpp

```diff
@@ -1,22 +1,1207 @@
+
 #include "mxdirectdraw.h"
 
-// OFFSET: LEGO1 0x1009e7f0
-int MxDirectDraw::FlipToGDISurface()
+//TODO: make commonn place for defines like that
+#ifndef _countof
+#define _countof(arr) sizeof(arr) / sizeof(arr[0])
```

Comment:
There's `legoutil.h` and `decomp.h` which would be suitable for macros like this